scripts/configure.py

```diff
@@ -316,6 +316,8 @@ def print_progress(tasks):
       return tasks
     # setup tmpfs (ram disk)
     tasks += print_progress(_setup_tmpfs(env['base_dir']))
+    # setup crontab
+    tasks += print_progress([Task("Setting up crontab", [f"cat {env['base_dir']}/scripts/crontab | sed 's@SCRIPTS_DIR@{env['base_dir']}/scripts@' | crontab"], shell=True).run()])
```

Comment:
should this go in AMPLIPI_DIR/config instead of AMPLIPI_DIR/scripts? I guess it is kinda a script, but it's not runnable which makes it an outlier in the scripts directory.